Documentation/DocuBlocks/Rest/Documents/delete_multiple_documents.md

```diff
@@ -83,10 +83,41 @@ is returned if `waitForSync` was `true`.
 @RESTRETURNCODE{202}
 is returned if `waitForSync` was `false`.
 
+@RESTRETURNCODE{403}
+is returned if the specified write concern for the collection cannot be
+fulfilled, the error code in this case is 1429. This can happen if less
+than the number of specified replicas for a shard are currently in sync
+are online. For example, if the write concern is 2 and the replication
```

Comment:
Nit: I find this sentence hard to understand, because of the two `are`s in it. Probably rephrase?

Reply:
Same comment applies to all following instances of this paragraph.

Reply:
Done. This was an oversight.